test/api_spec.js

```diff
@@ -927,7 +929,7 @@ describe("api", function () {
         public_id: 'test_folder1/test_subfolder2/item',
         tags: UPLOAD_TAGS,
       }),
-    ]).then(function (results) {
+    ]).then(wait(3000)).then(function (results) {
```

Comment:
Why do we need to wait here? 
If the upload returned, the assets supposed to be there

Reply:
We've encountered again and again that it's not the case.

There seems to be some very small, but non-zero delay from the upload success to the assets returning from an API call. 

We didn't encounter these issues as the account already had these assets in place

Reply:
Didn't we say will create a retry mechanism? or you want to keep for another task? if you do, please create a ticket.

Reply:
Yes it'll be another ticket to resolve these, we have a lot of them all over the code.